this is just to make Valley compile - that hack is not correct and needs to be fixed properly

--- Valley/src/Amalgam/VecAmalgam.cpp.org	2019-03-19 20:31:02.486426894 +0000
+++ Valley/src/Amalgam/VecAmalgam.cpp	2019-03-19 20:29:51.253023947 +0000
@@ -235,7 +235,7 @@
     __a32 = simde_mm_cvttps_epi32(simde_mm_mul_ps(__xDS, simde_mm_set1_ps(0x7FFFFFFF)));
     __b32 = simde_mm_cvttps_epi32(simde_mm_mul_ps(__yDS, simde_mm_set1_ps(0x7FFFFFFF)));
     __c32 = simde_mm_and_si128(simde_mm_or_si128(__c32, simde_mm_and_si128(__a32, __b32)), simde_mm_castps_si128(__chance32)); // Make 1 if a == 1 AND b == 1
-    __c32 = simde_mm_and_si128(__c32, ~simde_mm_and_si128(simde_mm_and_si128(~__a32, ~__b32), simde_mm_castps_si128(__chance32))); // Make 0 if a == 0 AND b == 0
+    __c32 = simde_mm_and_si128(__c32, simde_mm_and_si128(simde_mm_and_si128(__a32, __b32), simde_mm_castps_si128(__chance32))); // Make 0 if a == 0 AND b == 0
     __z = simde_mm_div_ps(simde_mm_cvtepi32_ps(__c32), simde_mm_set1_ps(0x7FFFFFFF));
     __zDS = simde_mm_switch_ps(__zDS, __z, __sample);
     return __zDS;
